assets/css/general.css

```diff
@@ -223,12 +268,41 @@ html[dir='rtl'] .clearButton {
     z-index: 10000000000000;
 }
 
+/* FontAwesome and Bootstrap 4 helpers */
+.fa.float-right { /* sass-lint:disable-line class-name-format */
+    margin-left: .3em;
+}
+
 /* RTL checking */
 
-html[dir='rtl'] .pull-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+html[dir='rtl'] body {
+    text-align: right !important; /* sass-lint:disable-line no-important */
+}
+
+html[dir='rtl'] .float-right:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
     float: left !important; /* sass-lint:disable-line no-important */
 }
 
-html[dir='rtl'] .pull-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
+html[dir='rtl'] .float-left:not(#rtlChecker) { /* sass-lint:disable-line class-name-format */
     float: right !important; /* sass-lint:disable-line no-important */
 }
+
+html[dir='rtl'] label input[type='checkbox'] {
+    margin-right: -20px;
+    margin-top: 5px;
+}
+
+html[dir='rtl'] button.close { /* sass-lint:disable-line no-qualifying-elements */
+    margin-left: 0 !important; /* sass-lint:disable-line no-important */
+}
+
+html[dir='rtl'] .btn-group > .btn:last-child:not(.dropdown-toggle), /* sass-lint:disable-line class-name-format */
+.btn-group > .btn-group:last-child > .btn { /* sass-lint:disable-line class-name-format */
+    border-radius: .2rem 0 0 .2rem;
+}
+
+
+html[dir='rtl'] .btn-group > .btn:first-child:not(.dropdown-toggle), /* sass-lint:disable-line class-name-format */
+.btn-group > .btn-group:first-child > .btn { /* sass-lint:disable-line class-name-format */
+    border-radius: 0 .2rem .2rem 0;
```

Comment:
I'm guessing that your change that fixed the alignment resulted in these styles no longer applying, breaking the round corners.

Reply:
Yeahh. Fixed.